presc/copies/copying.py

```diff
@@ -0,0 +1,55 @@
+from presc.copies.sampling import labeling
+from presc.copies.evaluations import empirical_fidelity_error
+
+
+class ClassifierCopy:
+    def __init__(
+        self,
+        original,
+        copy,
+        sampling_function,
+        *sampling_parameters,
+        **k_sampling_parameters
```

Comment:
I think it would be easier to use a single `**sampling_parameters`, meaning that any params need to be specified as kwargs. It makes the intent clearer and makes the code easier to maintain.

Reply:
Could be. In my mind I was thinking that users might want to implement their own sampling functions and that so that it might be interesting to give these sampling functions as much flexibility as possible.

Reply:
As discussed, I can't see a situation where someone implementing a custom sampling function would not be fine (or even prefer) using keyword args.